sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md

```diff
@@ -175,7 +175,7 @@ _The following was run on an Ubuntu Desktop 20.04 environment, with Arduino IDE
         Setting time using SNTP..............................done!
         Current time: Thu May 28 02:55:05 2020
         Client ID: mydeviceid
-        Username: myiothub.azure-devices.net/mydeviceid/?api-version=2018-06-30&DeviceClientType=c%2F1.0.0
+        Username: myiothub.azure-devices.net/mydeviceid/?api-version=2018-06-30&DeviceClientType=c%2F1.1.0-beta.1
```

Comment:
Since this isn't really a link, we should consider leaving it at the GA version rather than a beta one. Avoids us having to keep updating something that may not be what the user actually wants.